lambda/shared/src/main/scala/feral/lambda/LambdaEnv.scala

```diff
@@ -42,6 +42,15 @@ sealed trait LambdaEnv[F[_], Event] { outer =>
 object LambdaEnv {
   def apply[F[_], A](implicit env: LambdaEnv[F, A]): LambdaEnv[F, A] = env
 
+  def of[F[_]: Applicative, Event](e: Event, c: Context[F]): LambdaEnv[F, Event] =
+    new LambdaEnv[F, Event] {
+      val event = e.pure
+      val context = c.pure
+    }
```

Comment:
This is a sort of "escape-hatch" that lets you create a `LambdaEnv` if you have an `Event` and `Context`. This makes it possible to use the `LambdaEnv`-based middlewares even from `IOLambda.Simple`.

Reply:
On second thought, adding this API is a bit dangerous. In https://github.com/typelevel/feral/issues/85 I considered whether we may need additional methods on `LambdaEnv` to e.g. allow transformation of the `Event` in some way. In which case, an `Applicative` constraint would no longer be powerful enough to create a `LambdaEnv`.

Reply:
Reverted.